core/src/main/kotlin/pages/PageBuilder.kt

```diff
@@ -56,6 +57,16 @@ class DefaultPageBuilder(
 
     private fun contentForClass(c: Class) = group(c) {
         header(1) { text(c.name) }
+        if (c is Enum) {
+            header(2) { text("Entries") }
+            c.entries.forEach {
+                header(3) { text(it.name) }
+                val comment = it.commentsData.flatMap { it.children }
+                    .map { "${it.toHeaderString()}: ${it.root.docTagSummary()}" }.joinToString(separator = "\n")
+                text(comment)
+                text("\n")
```

Comment:
I'm almost sure we don't want to hardcode newlines, this should be achievable with groups or something similar